tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cpp

```diff
@@ -250,6 +255,163 @@ TEST (nnstreamer_nnfw_runtime_raw_functions, invoke)
   g_free (model_file);
 }
 
+/**
+ * @brief get argmax from the array
+ */
+size_t get_argmax (guint8 *array, size_t size)
+{
+  size_t idx, max_idx = 0;
+  guint8 max_value = 0;
+  for (idx = 0; idx < size; idx ++) {
+    if (max_value < array[idx]) {
+      max_idx = idx;
+      max_value = array[idx];
+    }
+  }
+
+  return max_idx;
+}
+
+/**
+ * @brief Test armnn subplugin with successful invoke for tflite advanced model
+ */
+TEST (nnstreamer_filter_armnn, invoke_advanced)
+{
+  int ret, fd;
+  void *data = NULL;
+  gchar *model_file, *manifest_file, *data_file;
+  const gchar *root_path = g_getenv ("NNSTREAMER_BUILD_ROOT_PATH");
+  const gchar *orig_model = "add.tflite";
+  const gchar *new_model = "mobilenet_v1_1.0_224_quant.tflite";
+  GstTensorMemory input, output;
+  GstTensorsInfo info, res;
+  char *replace_command;
+  ssize_t data_read;
+  size_t max_idx;
+  gboolean status;
+
+  ASSERT_NE (root_path, nullptr);
+
+  /** armnn needs a directory with model file and metadata in that directory */
+  model_file = g_build_filename (root_path, "tests", "test_models", "models",
+      "mobilenet_v1_1.0_224_quant.tflite", NULL);
+  status = g_file_test (model_file, G_FILE_TEST_EXISTS);
+  if (status == FALSE) {
+    g_free (model_file);
+    ASSERT_EQ (status, TRUE);
+  }
+
+  manifest_file = g_build_filename (root_path, "tests", "test_models", "models",
+      "metadata", "MANIFEST", NULL);
+  status = g_file_test (manifest_file, G_FILE_TEST_EXISTS);
+  if (status == FALSE) {
+    g_free (model_file);
+    g_free (manifest_file);
+    ASSERT_EQ (status, TRUE);
+  }
+
+  replace_command = g_strdup_printf("sed -i '/%s/c\\\"models\" : [ \"%s\" ],' %s",
```

Comment:
Is this line to use one MANIFEST file? 
That means... When we want to use multiple models, we have to use this way?
Maybe it related to issue #1981.

Reply:
This line modifies the MANIFEST file to tell the NNFW to use the desired model.
Yes, its related to #1981. The method used in the unittest is not the best way (this overwrites the complete line and assumes a specific format of the line previously), but works for this unittest. A better way would be to parse the MANIFEST file and then adjust it properly, which will be discussed in #1981.